src/main.scss

```diff
@@ -7,7 +7,8 @@
   --black:#212427;
   --nav-grey: #333333;
   --white: #ffffff;
-  --unactive-tabs: #636363;
+  --dashboard-grey: #484848;
+  --logo-colour: #542e71;
```

Comment:
The browser cannot read SASS so there's a compiler that converts the .scss file to the .css file so that the browser can read it. SASS just allows you to use more features and changes some syntax so that you can create dryer code